content/finalist/Tstar/index.md

```diff
@@ -0,0 +1,59 @@
+---
+type: "finalist"                   
+title: 
+name: "Tstar"
+description: "hello I am not a Bot Developer trust me"
+Team-Quote: "Making ideas a reality"
+Application_Type: ""
+---
+
+
+Hey ! We are Team Tstar.
```

Comment:
Atleast write 2 or 3 line to define your team that what are you all passionate about?

Reply:
Also in application Type mention.. Mention application type there.